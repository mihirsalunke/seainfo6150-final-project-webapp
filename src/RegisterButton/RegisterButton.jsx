import React from "react";
import styles from "./RegisterButton.module.css";

const RegisterButton = (props) => {
    return (
        <button className={styles.registerButton} onClick={props.onClick}>
            Register
        </button>
    );
};

export default RegisterButton;