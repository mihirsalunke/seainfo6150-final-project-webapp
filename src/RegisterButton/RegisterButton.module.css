@import url("https://fonts.googleapis.com/css2?family=Rowdies:wght@300&display=swap");

.registerButton {
    background: orange;
    border-radius: 10px;
    color: #fff;
    padding: 10px;
    font-family: Rowdies, sans-serif;
    font-size: 1.5em;
    margin: 0 0 20px;
    width: 5em;
    text-align: center;
    vertical-align: middle;

    transition: transform .25s ease-out;
}

.registerButton:hover {
    background: green;
    transform: translateY(2px);
    cursor: pointer;
}

.registerButton:active {
    transform: translateY(1px);
}