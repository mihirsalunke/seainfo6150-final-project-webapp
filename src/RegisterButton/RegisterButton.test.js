import React from "react";
import RegisterButton from "./RegisterButton.jsx";
import { render } from "@testing-library/react";

describe("RegisterButton tests", () => {
    it("renders correctly", () => {
      const { container } = render(<RegisterButton />);
      expect(container).toMatchSnapshot();
    });
});