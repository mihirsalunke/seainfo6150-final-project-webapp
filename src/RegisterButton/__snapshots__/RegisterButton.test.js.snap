// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterButton tests renders correctly 1`] = `
<div>
  <button
    class="registerButton"
  >
    Register
  </button>
</div>
`;
